bin/container.sh

```diff
@@ -237,9 +229,10 @@ console_mysql() {
 }
 
 status() {
-  eval $(docker-machine env ${project}) 2>/dev/null
-  docker-machine active 2>/dev/null | grep ${project} || usage "start ${project}" 1
-  eval $(docker-machine env ${project})
+  if [ "$OS" == "OSX" ]; then
+    docker-machine active 2>/dev/null | grep ${project} || usage "start ${project}" 1
```

Comment:
In a new shell, without running the "eval $(docker-machine ....)" statement first, it seems like "docker-machine active" command returns "no active host found", hence the "usage ..." conditional statement is always run.